bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+	"themes": "Teemat",
+	"prev-page": "Edellinen sivu",
+	"next-page": "Seuraava sivu",
+	"configure-plugin": "Lisäosan asetukset",
+	"confirm-delete-this-action-cannot-be-undone": "Haluatko varmasti poistaa? Tätä toimintoa ei voi perua.",
+	"site-title": "Sivuston otsikko",
+	"site-slogan": "Mainoslause",
+	"site-description": "Sivuston kuvaus",
+	"footer-text": "Alapalkin teksti",
+	"posts-per-page": "Blogikirjoituksia per sivu",
+	"site-url": "Sivuston URL-osoite",
+	"writting-settings": "Kirjoittamisen asetukset",
+	"url-filters": "Osoitteiden asetukset",
+	"page": "Sivu",
+	"pages": "Sivut",
+	"home": "Etusivu",
+	"welcome-back": "Tervetuloa takaisin",
+	"language": "Kieli",
+	"website": "Julkinen sivusto",
+	"timezone": "Aikavyöhyke",
+	"locale": "Maa/Kieli",
+	"new-post": "Uusi blogikirjoitus",
+	"new-page": "Uusi sivu",
+	"html-and-markdown-code-supported": "HTML sekä Markdown toimivat",
+	"manage-posts": "Hallitse blogikirjoituksia",
+	"published-date": "Julkaisuaika",
+	"modified-date": "Muokattu aika",
+	"empty-title": "Tyhjä otsikko",
+	"plugins": "Lisäosat",
+	"install-plugin": "Asenna lisäosa",
+	"uninstall-plugin": "Poista lisäosa",
+	"new-password": "Uusi salasana",
+	"edit-user": "Muokkaa käyttäjää",
+	"publish-now": "Julkaise nyt",
+	"first-name": "Etunimi",
+	"last-name": "Sukunimi",
+	"bludit-version": "Bludit:in versio",
+	"powered-by": "Virtaa antaa: ",
+	"recent-posts": "Viimeisimmät blogikirjoitukset",
+	"manage-pages": "Hallitse sivuja",
+	"advanced-options": "Lisäasetukset",
+	"user-deleted": "Käyttäjä poistettu",
+	"page-added-successfully": "Sivu lisätty onnistuneesti",
+	"post-added-successfully": "Blogikirjoitus lisätty onnistuneesti",
+	"the-post-has-been-deleted-successfully": "Blogikirjoitus poistettu",
+	"the-page-has-been-deleted-successfully": "Sivu poistettu",
+	"username-or-password-incorrect": "Käyttäjätunnus tai salasana virheellinen",
+	"database-regenerated": "Tietokanta uudelleengeneroitu",
+	"the-changes-have-been-saved": "Muutokset on tallennettu",
+	"enable-more-features-at": "Ota käyttöön lisää ominaisuuksia",
+	"username-already-exists": "Käyttäjätunnus on jo olemassa",
+	"username-field-is-empty": "Käyttäjätunnus on tyhjä",
+	"the-password-and-confirmation-password-do-not-match":"Salasana ja sen vahvistus eivät täsmää",
+	"user-has-been-added-successfully": "Käyttäjä lisätty onnistuneesti",
+	"you-do-not-have-sufficient-permissions": "Sinulla ei ole tarvittavia oikeuksia tälle sivulle. Ota yhteyttä ylläpitäjään.",
+	"settings-advanced-writting-settings": "Asetukset->Lisäasetukset->Kirjoittamisen asetukset",
+	"new-posts-and-pages-synchronized": "Uudet sivut ja blogikirjoitukset synkronisoitu.",
+	"you-can-choose-the-users-privilege": "Voit valita käyttäjän tyypin. Kirjoittaja voi ainostaan muokata sivuja ja blogikirjoituksia, ylläpitäjä myös muuttaa asetuksia.",
+	"email-will-not-be-publicly-displayed": "Sähköpostia ei näytetä julkisesti. Suositeltu salasanan palauttamiseen ja ilmoituksiin.",
```

Comment:
"email-will-not-be-publicly-displayed": "Sähköpostia ei näytetä julkisesti. Suositellaan salasanan palauttamiseen ja ilmoituksiin.",